examples/05_deduplication.py

```diff
@@ -136,43 +137,44 @@
 plt.ylabel("Counts")
 
 ###############################################################################
-# In this example we can correct all spelling mistakes by using the ideal number
-# of clusters as determined by the silhouette score.
+# Here, the silhouette score finds the ideal number of
+# clusters and groups the spelling mistakes.
 #
-# However, often the translation/deduplication won't be perfect and will require some
-# tweaks. In this case, we can construct and update a translation table based on the
-# data returned by |dd|.
-# It consists of the (potentially) misspelled category names as indices and the
-# (potentially) correct categories as values.
+# In practice, the translation/deduplication will often be imperfect
+# and require some tweaks.
+# In this case, we can construct and update a translation table based on the
+# data returned by |deduplicate|.
```

Comment:
Maybe move the plotting code right above after this code that shows how to apply the deduplication, as applying the deduplication is probably what the busy user is looking for.

Also, maybe do 2 different subsections.